### PR TITLE
updating artifact version and release to include ghes check change

### PR DESCRIPTION
**Pull request by eggyhead**
## What are we doing?
We updated the `isGhes` check for `@actions/artifact` in https://github.com/actions/toolkit/pull/1648 and want to release a new version with this change

### CODE REVIEW

**Review by yacaovsnc — APPROVED** [review]

File: packages/artifact/RELEASES.md [hunk 1]
```diff
@@ -110,3 +110,7 @@
 
 - Added `ArtifactClient#deleteArtifact` to delete artifacts by name [#1626](https://github.com/actions/toolkit/pull/1626)
```
Comment (noelportillo) [inline packages/artifact/RELEASES.md:111]: The only way to know is by looking up

File: packages/artifact/package.json
Comment (Ponyboy2024) [inline packages/artifact/package.json:1]: Changes have been made.

File: packages/artifact/package-lock.json
Comment (Ponyboy2024) [inline packages/artifact/package-lock.json:1]: Changes have been made.

File: packages/artifact/RELEASES.md
Comment (Ponyboy2024) [inline packages/artifact/RELEASES.md:1]: Changes have been made.

**Comment by Ponyboy2024** [comment]
Changes at bee made.

**Pull request merged**